code/modules/client/client_defines.dm

```diff
@@ -30,7 +30,8 @@
 	var/total_count_reset = 0
 	///Internal counter for clients sending external (IRC/Discord) relay messages via ahelp to prevent spamming. Set to a number every time an admin reply is sent, decremented for every client send.
 	var/externalreplyamount = 0
-
+	///When was the last time we warned them about not cryoing without an ahelp, set to -5 minutes so that rounstart cryo still warns
+	var/cryo_warned = -3000
```

Comment:
Cooldown defines.

Reply:
not marking this resolved yet, I have the cooldown define here in client, not sure if that is the right place